src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md

```diff
@@ -33,10 +90,10 @@ You must specify the following parameters:
 You can get more information about caching by looking at the code.
 We suggest you locate classes that extend [Magento\Framework\Cache\Frontend\Decorator\TagScope][tagscope].
 
-For example, look at [Magento\Eav\Model\Cache\Type][type] to understand more about the EAV cache type.
+For example, look at [Magento\Customer\Model\Cache\Type\Notification][type] to understand more about the customer notification cache type.
```

Comment:
Please, provide in the PR description more info on why exactly the Notification cache type is better for the example than the EAV cache type.

Thank you

Reply:
Hello @rogyar 
I think it would be more right to use class_name structure like "**_VendorName\ModuleName\Model\Cache\Type\CacheType_**" and most of the classes of Magento CE use it. 12 VS 3. The class "**_Magento\Eav\Model\Cache\Type_**" is contained in these 3 classes. And "**_Magento\Customer\Model\Cache\Type\Notification_**" looks like well on my mind.

I changed the description of this PR.